### PR TITLE
RavenDB-14699 Expose LoadRequestTimeoutInSec configuration option for…

### PR DESCRIPTION
**Pull request by Danielle9897**
… Raven ETL task

### CODE REVIEW

File: src/Raven.Studio/wwwroot/App/views/database/tasks/editRavenEtlTask.html [hunk 1]
```diff
@@ -95,6 +95,26 @@ <h3 data-bind="text: $parent.isAddingNewRavenEtlTask() ? `New RavenDB ETL` : `Ed
...
+                                </button>
+                            </div>
+                        </div>
+                        <div data-bind="collapse: $root.showAdvancedOptions">
+                            <div class="panel-body">
+                                <div class="form-group" data-bind="validationElement: loadRequestTimeout">
+                                    <label for="loadTimeout" class="control-label">Http Request Timeout</label>
+                                    <div class="flex-grow">
+                                        <div class="input-group">
+                                            <input type="number" min="0" class="form-control" placeholder="Use HttpClient default timeout (100s)" id="LoadTimeout" data-bind="numericInput: loadRequestTimeout" autocomplete="off">
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/App/views/database/tasks/editRavenEtlTask.html]: for and id are not bound correctly - loadTimeout vs LoadTimeout

Comment (ml054) [reply src/Raven.Studio/wwwroot/App/views/database/tasks/editRavenEtlTask.html]: Do we need to say that default timeout is HttpClient timeout?

Comment (Danielle9897) [reply src/Raven.Studio/wwwroot/App/views/database/tasks/editRavenEtlTask.html]: 1. Done - unified for and id
2. Probably can do without explicitly saying 'HttpClient' and just say: '_Use default timeout (100s_)'
    I just needed to have some text similar to the sql-etl task where we have: '_Use SQL provider default value_'

Comment (arekpalinski) [reply src/Raven.Studio/wwwroot/App/views/database/tasks/editRavenEtlTask.html]: 2. Yeah, we can drop 'HttpClient' there

**Review by ml054 — APPROVED** [review]

**Pull request merged**